src/examples.json

```diff
@@ -1,1328 +1,1742 @@
 [
     {
+        "$schema": "https://json-schema.org/draft/2020-12/schema",
         "type": "object",
         "properties": {},
-        "required": [],
-        "additionalProperties": false,
-        "$schema": "https://json-schema.org/draft/2020-12/schema"
+        "additionalProperties": false
     },
     {
+        "$schema": "https://json-schema.org/draft/2020-12/schema",
         "type": "object",
         "properties": {
             "tab": {
                 "type": "string"
             }
         },
-        "required": ["tab"],
-        "additionalProperties": false,
-        "$schema": "https://json-schema.org/draft/2020-12/schema"
+        "required": [
+            "tab"
+        ],
+        "additionalProperties": false
     },
     {
+        "$schema": "https://json-schema.org/draft/2020-12/schema",
         "type": "object",
         "properties": {
             "table": {
                 "type": "string"
             }
         },
-        "required": ["table"],
-        "additionalProperties": false,
-        "$schema": "https://json-schema.org/draft/2020-12/schema"
+        "required": [
+            "table"
+        ],
+        "additionalProperties": false
     },
     {
+        "$schema": "https://json-schema.org/draft/2020-12/schema",
         "type": "object",
         "properties": {
             "mode": {
-                "enum": ["Dashboard", "Data", "Layout", "Workflows", "Settings"]
+                "type": "string",
+                "enum": [
```

Comment:
Any chance you could undo the reformatting, to make it easier to review the changes?

Reply:
Apologies, I had an auto format rule running in my IDE. I have updated to only show the changes.

Most of the changes in the example file bring it inline with the expected output from Zod's toJson functionality.